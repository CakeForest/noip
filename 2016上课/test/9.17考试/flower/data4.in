100 100
2 26 71
1 12 70
2 35 99
1 11 4
1 73 34
1 11 42
2 47 68
1 57 63
2 23 59
2 29 78
1 90 36
1 6 89
1 64 43
1 5 47
1 70 30
1 1 7
2 29 48
2 54 84
1 66 41
1 8 32
1 26 40
2 37 38
1 29 83
2 15 33
2 4 58
1 6 78
2 21 86
2 24 72
1 77 30
2 12 97
1 61 91
1 67 56
2 31 74
1 50 51
2 24 66
1 91 8
2 37 57
2 53 83
2 9 9
1 88 22
1 6 47
1 68 14
1 62 92
2 10 59
1 48 38
2 41 95
1 91 51
1 20 75
1 48 22
2 68 84
2 34 53
2 18 38
1 27 89
2 28 93
1 7 84
2 10 17
2 9 14
1 51 36
1 19 50
1 3 99
1 44 9
2 2 89
2 85 93
2 23 87
1 48 4
1 18 59
1 98 97
2 89 98
2 57 72
1 92 39
1 90 80
2 58 91
2 56 88
2 2 34
1 28 56
1 86 63
2 33 69
1 16 45
2 22 98
2 21 99
2 76 92
2 12 75
1 3 11
2 61 88
2 55 89
2 2 85
1 88 86
1 57 18
1 69 33
1 89 22
1 68 30
1 55 26
1 41 50
1 60 46
1 39 54
2 79 96
2 29 49
2 49 66
2 95 97
1 5 87
