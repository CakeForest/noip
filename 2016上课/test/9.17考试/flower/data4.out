0
12 81
47
11 37
73 99
38 72
22
57 68
27
19
Can not put any one.
6 78
Can not put any one.
5 5
Can not put any one.
1 4
20
31
66 84
29 65
Can not put any one.
2
37 38
19
36
6 58
66
0
77 86
30
61 97
Can not put any one.
14
50 74
17
Can not put any one.
0
17
1
Can not put any one.
9 57
68 81
62 83
48
48 61
48
91 95
20 90
Can not put any one.
17
20
14
27 84
66
10 93
8
1
Can not put any one.
Can not put any one.
4 17
Can not put any one.
88
4
0
48 51
18 80
Can not put any one.
5
16
92 98
90 91
10
1
17
28 91
Can not put any one.
37
16 65
73
2
0
5
3 13
0
0
11
88 99
57 74
75 87
Can not put any one.
Can not put any one.
55 56
41 54
Can not put any one.
39 40
18
11
17
1
5 97
